critiquebrainz/frontend/static/robots.txt

```diff
@@ -0,0 +1,7 @@
+User-agent: *
+Disallow: /admin
```

Comment:
I saw an admin folder and thought something like this might exist on the server.
I'll remove this.